---a/net/minecraft/command/server/CommandTestFor.java
+++b/net/minecraft/command/server/CommandTestFor.java
@@ -1,6 +1,7 @@
 package net.minecraft.command.server;
 
 import java.util.List;
+
 import net.minecraft.command.CommandBase;
 import net.minecraft.command.CommandException;
 import net.minecraft.command.ICommandSender;
@@ -12,71 +12,56 @@
 import net.minecraft.server.MinecraftServer;
 import net.minecraft.util.BlockPos;
 
-public class CommandTestFor extends CommandBase
-{
+public class CommandTestFor extends CommandBase {
+
     private static final String __OBFID = "CL_00001182";
 
-    public String getName()
-    {
+    public String getName() {
         return "testfor";
     }
 
-    public int getRequiredPermissionLevel()
-    {
+    public int getRequiredPermissionLevel() {
         return 2;
     }
 
-    public String getCommandUsage(ICommandSender sender)
-    {
+    public String getCommandUsage(ICommandSender sender) {
         return "commands.testfor.usage";
     }
 
-    public void execute(ICommandSender sender, String[] args) throws CommandException
-    {
-        if (args.length < 1)
-        {
+    public void execute(ICommandSender sender, String[] args) throws CommandException {
+        if (args.length < 1) {
             throw new WrongUsageException("commands.testfor.usage", new Object[0]);
-        }
-        else
-        {
+        } else {
             Entity entity = func_175768_b(sender, args[0]);
             NBTTagCompound nbttagcompound = null;
 
-            if (args.length >= 2)
-            {
-                try
-                {
+            if (args.length >= 2) {
+                try {
                     nbttagcompound = JsonToNBT.func_180713_a(func_180529_a(args, 1));
+                } catch (NBTException nbtexception) {
+                    throw new CommandException("commands.testfor.tagError", new Object[]{nbtexception.getMessage()});
                 }
-                catch (NBTException nbtexception)
-                {
-                    throw new CommandException("commands.testfor.tagError", new Object[] {nbtexception.getMessage()});
-                }
             }
 
-            if (nbttagcompound != null)
-            {
+            if (nbttagcompound != null) {
                 NBTTagCompound nbttagcompound1 = new NBTTagCompound();
                 entity.writeToNBT(nbttagcompound1);
 
-                if (!CommandTestForBlock.func_175775_a(nbttagcompound, nbttagcompound1, true))
-                {
-                    throw new CommandException("commands.testfor.failure", new Object[] {entity.getName()});
+                if (!CommandTestForBlock.func_175775_a(nbttagcompound, nbttagcompound1, true)) {
+                    throw new CommandException("commands.testfor.failure", new Object[]{entity.getName()});
                 }
             }
 
-            notifyOperators(sender, this, "commands.testfor.success", new Object[] {entity.getName()});
+            notifyOperators(sender, this, "commands.testfor.success", new Object[]{entity.getName()});
         }
     }
 
-    public boolean isUsernameIndex(String[] args, int index)
-    {
+    public boolean isUsernameIndex(String[] args, int index) {
         return index == 0;
     }
 
-    public List addTabCompletionOptions(ICommandSender sender, String[] args, BlockPos pos)
-    {
+    public List addTabCompletionOptions(ICommandSender sender, String[] args, BlockPos pos) {
         return args.length == 1 ? getListOfStringsMatchingLastWord(args, MinecraftServer.getServer().getAllUsernames()) : null;
     }
-}
+}
